openlayers/index.d.ts

```diff
@@ -6669,20 +6667,22 @@ declare module ol {
          *     the {@link ol.layer.Layer layer} of the feature and will be null for
          *     unmanaged layers. To stop detection, callback functions can return a
          *     truthy value.
-         * @param {S=} opt_this Value to use as `this` when executing `callback`.
          * @param {(function(this: U, ol.layer.Layer): boolean)=} opt_layerFilter Layer
          *     filter function. The filter function will receive one argument, the
          *     {@link ol.layer.Layer layer-candidate} and it should return a boolean
          *     value. Only layers which are visible and for which this function returns
          *     `true` will be tested for features. By default, all visible layers will
          *     be tested.
-         * @param {U=} opt_this2 Value to use as `this` when executing `layerFilter`.
          * @return {T|undefined} Callback result, i.e. the return value of last
          * callback execution, or the first truthy callback return value.
-         * @template S,T,U
+         * @template T
          * @api stable
          */
-        forEachFeatureAtPixel<S, T, U>(pixel: ol.Pixel, callback: ((feature: (ol.Feature | ol.render.Feature), layer: ol.layer.Layer) => T), opt_this?: S, opt_layerFilter?: ((layer: ol.layer.Layer) => boolean), opt_this2?: U): (T);
+        forEachFeatureAtPixel<T>(
```

Comment:
hitTolerance only appeared in v3.20.0. The modification here refers to v3.18.0. Since then, the definition file has been updated to v3.20 and this PR is outdated.